site/blog/2019/sep/19/helping-first-time-conference-attendees-with-welcome-wagon.rst

```diff
@@ -0,0 +1,87 @@
+Helping First-Time Conference Attendees with the Welcome Wagon
+==============================================================
+
+You walk into the room and you're wearing the *wrong* thing.
+Everyone else is dressed casually,
+but you are not.
+You feel silly and find the quickest excuse to leave the situation.
+This is an example of a common situation:
+not knowing what is expected of you when you enter a new place.
+
+It's stressful to enter a new group without knowing what to expect.
+It breeds anxiety and self-doubt,
+and makes it much more likely you'll find excuses not to join.
+
+At Write the Docs,
+we're working to reduce the possibility of this happening.
+We have a program that we call the Welcome Wagon,
+that helps attendees at both the planning and the attending stage of the conference.
+We hope that this makes it much easier for them to interact and get the most value out of being in our community.
+
+Welcome Wagon Guide
+-------------------
+
+One major part of the Welcome Wagon is our `guide`_.
+It acts as documentation on our community,
+a User Guide for the conference.
+It covers:
+
+* Common questions, like "How should I dress?" and "Where is everything?"
+* How to plan your time at the conference
+* What is expected of you while you're at the conference
+* Tips and tricks for having the best time at the event
+
+We answer attendees most common questions before the event,
+making it more explicit what is expected of them and how they can make sure to enjoy themselves.
+This reduces anxiety and also makes it more likely that people will have a good first experience with our community.
+
+.. _guide: https://www.writethedocs.org/conf/portland/2019/welcome-wagon/
+
+Welcome Wagon Events
+--------------------
+
+The second major part of the Welcome Wagon is a set of events at the conference itself.
+We have a team who are the humans that are the "Welcome Wagon",
+and they lead the events at the conference.
+Their picture is also in the guide to make it easy to find them at the event.
+
+Currently we do:
+
+* An introduction the evening before the conference
+* Venue tours the evening before and morning of the conference
+* We also invite the attendees to ask the Welcome Wagon team questions any time throughout the conference
+
+The goal of these events is to introduce the Welcome Wagon team to people,
+as well as to explain how the conference and community works.
+Attendees will ask the Welcome Wagon team questions throughout the conference,
+because they know their faces and that they will be helpful.
+
+Run a Welcome Wagon
+-------------------
+
+If you're curious about how to run a Welcome Wagon at your conference,
+you're in luck.
+We have a `organizer guide`_ that explains how to do it.
```

Comment:
It would be cool to have this guide and the welcome wagon as docs in a repo so people could just fork it easily.

Reply:
Agreed! I wonder the best path forward on it. The PyCascades folks made it a specific repo for each event, which might work but is also a bit heavy :/